SIGSEGV with `pixlet check`

```
❯ pixlet check christmascountdown.star
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x50 pc=0x10584861c]

goroutine 1 [running]:
github.com/tronbyt/pixlet/cmd.render(0x106e81d60, {0x14000517c20, 0x1, 0x1})
	github.com/tronbyt/pixlet/cmd/render.go:186 +0x3ec
github.com/tronbyt/pixlet/cmd.checkCmd(0x106e81d60, {0x14000305650?, 0x4?, 0x1058a4c61?})
	github.com/tronbyt/pixlet/cmd/check.go:105 +0x3ac
github.com/spf13/cobra.(*Command).execute(0x106e81d60, {0x14000305620, 0x1, 0x1})
	github.com/spf13/cobra@v1.10.1/command.go:1015 +0x7d4
github.com/spf13/cobra.(*Command).ExecuteC(0x106e7fdc0)
	github.com/spf13/cobra@v1.10.1/command.go:1148 +0x350
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.10.1/command.go:1071
main.main()
	github.com/tronbyt/pixlet/main.go:44 +0x24
❯ pixlet version
Pixlet version: 0.48.1
```

--- a/cmd/render.go
+++ b/cmd/render.go
@@ -183,7 +183,7 @@ func render(cmd *cobra.Command, args []string) error {
 	case "webp":
 		imageFormat = loader.ImageWebP
 		outPath += ".webp"
-		if cmd.Flags().Lookup(webpLevelFlag).Changed {
+		if flag := cmd.Flags().Lookup(webpLevelFlag); flag != nil && flag.Changed {
 			encode.SetWebPLevel(webpLevel)
 		}
 	case "gif":

--- a/cmd/serve.go
+++ b/cmd/serve.go
@@ -64,7 +64,7 @@ func serve(cmd *cobra.Command, args []string) error {
 	switch serveFormat {
 	case "webp":
 		imageFormat = loader.ImageWebP
-		if cmd.Flags().Lookup(webpLevelFlag).Changed {
+		if flag := cmd.Flags().Lookup(webpLevelFlag); flag != nil && flag.Changed {
 			encode.SetWebPLevel(webpLevel)
 		}
 	case "gif":

